test/pummel/test-heapsnapshot-near-heap-limit-by-api.js

```diff
@@ -0,0 +1,131 @@
+// Copy from test-heapsnapshot-near-heap-limit.js
+'use strict';
+
+const common = require('../common');
+
+if (common.isPi) {
+  common.skip('Too slow for Raspberry Pi devices');
+}
+
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+const v8 = require('v8');
+
+if (process.env.isChild === '1') {
+  require(fixtures.path('workload', 'grow.js'));
+  const count = ~~process.env.count || 1;
+  for (let i = 0; i < count; i++) {
+    v8.setHeapSnapshotNearHeapLimit(+process.env.limit);
```

Comment:
It's unlikely that the loading of `grow.js` would immediately lead to a heap size near the limit, but still it's probably safer to run this before loading the workload.

Reply:
Done !